srcpkgs/faust/patches/faust-voidlinux-lib_and_arch_paths.patch

```diff
@@ -0,0 +1,34 @@
+--- compiler/libcode.cpp	2019-12-11 10:32:13.000000000 +0000
++++ compiler/libcode.cpp-new	2019-12-19 07:17:01.675119720 +0000
+@@ -733,11 +733,11 @@
+ }
+ static void printArchDir()
+ {
+-    cout << gGlobal->gFaustRootDir << kPSEP << "share" << kPSEP << "faust" << endl;
++    cout << gGlobal->gFaustRootDir << kPSEP << "lib" << kPSEP << "faust" << endl;
+ }
+ static void printDspDir()
+ {
+-    cout << gGlobal->gFaustRootDir << kPSEP << "share" << kPSEP << "faust" << endl;
++    cout << gGlobal->gFaustRootDir << kPSEP << "lib" << kPSEP << "faust" << endl;
+ }
+ static void printPaths()
+ {
+@@ -1101,6 +1101,8 @@
+     gGlobal->gImportDirList.push_back(exepath::dirup(gGlobal->gFaustExeDir) + "/share/faust");
+     gGlobal->gImportDirList.push_back("/usr/local/share/faust");
+     gGlobal->gImportDirList.push_back("/usr/share/faust");
++    gGlobal->gImportDirList.push_back("/usr/local/lib/faust");
++    gGlobal->gImportDirList.push_back("/usr/lib/faust");
+ 
+     //-------------------------------------------------------------------------------------
+     // init gArchitectureDirList : a list of path where to search architectures files
+@@ -1121,6 +1123,8 @@
+     gGlobal->gArchitectureDirList.push_back("/usr/share/faust");
+     gGlobal->gArchitectureDirList.push_back("/usr/local/include");
+     gGlobal->gArchitectureDirList.push_back("/usr/include");
++    gGlobal->gArchitectureDirList.push_back("/usr/local/lib/faust");
```

Comment:
are you sure you want it to be `/usr/local/`? The package shouldn't install anything in there...